### ISSUE TITLE
Missing letters in saved pdf document.

### ISSUE THREAD

**Issue by wz93672** [pymupdf/PyMuPDF#3362]
### Description of the bug

I have two problems with pdf documents which I’m getting from outside and don’t have influence on documents creation. This one is missing letters in pdf opened and saved with pymupdf. There was some inconsistency with replication, but looks like “clean = True” option for save method is triggering the bug. In the sample it is visible in red headline.

![image](https://github.com/pymupdf/PyMuPDF/assets/166443955/c4667126-492b-4747-93a8-3f83756a7076)

[default options_first save.pdf](https://github.com/pymupdf/PyMuPDF/files/14910510/default.options_first.save.pdf)
[test1_clean-True_first save.pdf](https://github.com/pymupdf/PyMuPDF/files/14910516/test1_clean-True_first.save.pdf)

### How to reproduce the bug

```python
with fitz.open('default options_first save.pdf') as doc:
   doc.save('test1_clean-True_first save.pdf', clean=True)
```

### PyMuPDF version

1.24.1

### Operating system

Windows

### Python version

3.11

**Comment by JorjMcKie** [comment]
This is an error in the base software. 
It also is exhibited when using MuPDF's CLI tool:
```bash
mutool clean -s default.options_first.save.pdf
warning: dropping unclosed PDF processor
warning: ... repeated 2 times...
```
I have submitted a report in its issue system with a link to here.
MuPDF bug report link: https://bugs.ghostscript.com/show_bug.cgi?id=707727

**Comment by JorjMcKie** [comment]
This has been fixed in v1.24.2.
